src/js/forecast.js

```diff
@@ -17,12 +17,15 @@ function getForecast() {
         return response.json();
       })
       .then((data) => {
-        tempForecastDay1.innerHTML = Math.round(data.daily[1].temp.day);
-        tempForecastDay2.innerHTML = Math.round(data.daily[2].temp.day);
-        tempForecastDay3.innerHTML = Math.round(data.daily[3].temp.day);
+        function round(temp) {
+          return Math.round(temp);
+        }
```

Comment:
Good idea would be to separate with to 
`helperFunction.js` file (name could be different) and import it and use it.
Modularity is the better approach usually.